ray-operator/controllers/ray/common/pod.go

```diff
@@ -225,21 +229,35 @@ func convertCmdToString(cmdArr []string) (cmd string) {
 	return cmdAggr.String()
 }
 
-func getRayContainerIndex(pod v1.Pod) (index int) {
-	// theoretically, a ray pod can have multiple containers.
+func getRayContainerIndex(pod v1.Pod) (rayContainerIndex int) {
+	// a ray pod can have multiple containers.
 	// we identify the ray container based on env var: RAY=true
 	// if the env var is missing, we choose containers[0].
 	for i, container := range pod.Spec.Containers {
 		for _, env := range container.Env {
 			if env.Name == strings.ToLower("ray") && env.Value == strings.ToLower("true") {
+				log.Info("Head pod container with index " + strconv.Itoa(i) + " identified as Ray container based on env RAY=true.")
 				return i
 			}
 		}
 	}
 	// not found, use first container
+	log.Info("Head pod container with index 0 identified as Ray container.")
```

Comment:
I think it would make sense to document the requirement that the Ray container goes first.
To me, seems odd to identify a container by an env variable -- also, none of the sample configs I've seen do that.